lib/vsdata.cpp

```diff
@@ -47,6 +47,104 @@ void VisualizationSceneScalarData::FixValueRange()
    }
 }
 
+int VisualizationSceneScalarData::GetFunctionAutoRefineFactor(GridFunction &gf)
+{
```

Comment:
This is too complicated and wrong in my opinion. We just need to get the max order from the space and the max order from the mesh.

Reply:
That is what it does, but unfortunately different visualization objects have differently named grid functions.

Reply:
I'm talking about the checks for integral map type, the checks for volume vs boundary, checks for discontinuous element types -- all of that is unnecessary -- just get `GetMaxElementOrder()` and multiply it by 2 (I'm not sure we want that multiplication by 2 actually).

Reply:
As said below, I think we may fly with the formula `2*order-1`, which is a bit more conservative.